relentless/simulate/lammps.py

```diff
@@ -512,107 +506,87 @@ def to_commands(self, sim):
 
         return cmds
 
-class AddNPTIntegrator(LAMMPSOperation):
-    r"""NPT integration via Nos\'e-Hoover thermostat/barostat.
+class Thermostat:
+    def __init__(self, T):
+        self.T = T
 
-    Parameters
-    ----------
-    dt : float
-        Time step size for each simulation iteration.
-    tau_T : float
-        Coupling constant for the thermostat.
-    tau_P : float
-        Coupling constant for the barostat.
+class BerendsenThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
 
-    """
-    def __init__(self, dt, tau_T, tau_P):
-        self.tau_T = tau_T
-        self.tau_P = tau_P
+class NoseHooverThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
 
-        self._fix = super().new_fix_id()
+class MTKThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
 
-    def to_commands(self, sim):
-        cmds = ['fix {idx} {group_idx} npt temp {Tstart} {Tstop} {Tdamp} iso {Pstart} {Pstop} {Pdamp}'.format(idx=self._fix,
-                                                                                                              group_idx='all',
-                                                                                                              Tstart=sim.ensemble.T,
-                                                                                                              Tstop=sim.ensemble.T,
-                                                                                                              Tdamp=self.tau_T,
-                                                                                                              Pstart=sim.ensemble.P,
-                                                                                                              Pstop=sim.ensemble.P,
-                                                                                                              Pdamp=self.tau_P)]
+class Barostat:
+    def __init__(self, P):
+        self.P = P
 
-        return cmds
+class MTKBarostat(Barostat):
+    def __init__(self, P, tau):
+        super().__init__(P)
+        self.tau = tau
 
```

Comment:
There is, however, a `BerendsenBarostat` that works the same way as the thermostat. We should add this, as Beth used it in one of her papers.